fuzz/fuzz_targets/header_length.rs

```diff
@@ -1,21 +1,15 @@
 #![no_main]
 use libfuzzer_sys::fuzz_target;
-use mongodb::cmap::conn::wire::header::Header;
+use mongodb::cmap::conn::wire::{header::Header, message::Message};
 
 fuzz_target!(|data: &[u8]| {
     if data.len() < Header::LENGTH {
         return;
     }
     if let Ok(header) = Header::from_slice(data) {
-        if header.length < 0 {
```

Comment:
This was particularly bad. Ok, the length is negative because it's random. This wasn't testing any actual driver code :joy: